tools/clang_format_utils.py

```diff
@@ -90,7 +90,7 @@ def get_and_check_clang_format(verbose=False):
         # If the directory doesn't exist, try to create it.
         try:
             os.mkdir(CLANG_FORMAT_DIR)
-        except os.OSError as e:
+        except OSError as e:
```

Comment:
why did this have the `os.` prefix before? I don't see a `from os import OSError` in the file currently so I'm a bit confused

Reply:
I don't know, found those while running mypy on the file.